dev/com.ibm.ws.microprofile.rest.client.FT_fat/test-applications/retryApp/resources/WEB-INF/beans.xml

```diff
@@ -1,9 +1,4 @@
-<beans
-   xmlns="http://java.sun.com/xml/ns/javaee"
-   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xsi:schemaLocation="
-      http://java.sun.com/xml/ns/javaee
-      http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
+<beans>
```

Comment:
Strictly neither the old or new versions are valid beans.xml
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://java.sun.com/xml/ns/javaee"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
```

Reply:
I based the changes on this article:
https://blog.payara.fish/getting-started-with-jakarta-ee-9-context-and-dependency-injection-cdi

In the "interceptors" section, Rudy De Busscher writes:
```
<beans>
   <interceptors>
      <class>fish.payara.jakarta.ee9.start.TimedInterceptor</class>
    </interceptors>
</beans>
```
> In the above example, we did not specify the namespace of the beans tag. This has the advantage that it can be used by different versions of Java EE and Jakarta EE (since the namespace changes between Java EE, Jakarta EE 8, and Jakarta EE 9) but we do not specify the bean discovery mode.

Please let me know if there is a better approach.

Reply:
I suspect that article was really just using snippets. Without the xml header and namespaces etc, it might still work but it is not _strictly_ correct. Whether is works or not would be implementation specific.

Reply:
If you are seeing other issues then is is possible that the file is just being silently ignored??